examples/advanced/train_act_pusht/train_act_pusht.md

```diff
@@ -0,0 +1,62 @@
+In this tutorial we will adapt the default configuration for ACT to be compatible with the PushT environment and dataset.
```

Comment:
I think this tutorial is too difficult to follow. It's a good start, but we should iterate to simplify as much as possible.

Ideally we should just provide the script to train act on pusht, and comment as much as possible.
And also to provide the script to train diffusion policy on aloha.

Reply:
Unfortunately, the `yaml` file is necessary because the observation key changes via the CLI are challenging due to the "." separator and the nested lists. So we'd need two files at least.

I like the MD file as it allows for a more tutorial style voice and structure.

What do you think we could do to make it easier to follow?

Why should we also provide the script for DP / Aloha? What value does it add over one example that gets the point across?

Reply:
I dont know exactly know, but we should try to compress the information. 
When it's too long, people dont read.

I am just sharing high level thoughts sorry ^^'

Reply:
I've had a read through again and couldn't find any obvious ways to cut it. How about we leave it to user testing (aka merge it and see how people interact with it).